composer.json

```diff
@@ -25,7 +27,10 @@
         "ext-mbstring": "*",
         "ext-curl": "*",
         "composer/ca-bundle": "~1.0",
-        "symfony/options-resolver": "~2.7|~3.0"
+        "symfony/options-resolver": "~2.7|~3.0",
+        "php-http/httplug": "~1.1",
+        "php-http/curl-client": "~1.7",
+        "zendframework/zend-diactoros": "~1.4"
```

Comment:
I've updated the `composer.json` dependencies, moved around the `require` and `require-dev` sections (I know this was not something related to this PR but I wanted to do it since a long time) and I followed your suggestion to not require a specific HTTP client implementation and instead rely on Composer virtual packages